ipm/README

```diff
@@ -0,0 +1,22 @@
+This samples shows a simple communication between two cores
+on Zephyr. It can be tested on an Arduino 101, for instance.
+This board have two cores: one ARC and one Quark (x86).
+Directories `x86/` and `arc/` have program that runs on x86 and ARC
+core, respectively.
```

Comment:
Fine, but I'd state what it really does: sends strings from both sides indefinitely.